pkg/helm/values.go

```diff
@@ -0,0 +1,21 @@
+package helm
+
+import (
+	"strings"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+type HelmValues map[string]interface{}
+
+func (h HelmValues) GetString(key string) (string, bool, error) {
+	return unstructured.NestedString(h, toKeys(key)...)
+}
+
+func (h HelmValues) Set(key string, val string) error {
```

Comment:
Should this be called `SetString`?

Reply:
Ah. No, `Set` is fine, but the second argument should be `val interface{}`. Thanks!

Reply:
Fixed.